kits/projectpilot-ai/prompts/execution-flow_instructor-llmnode-603_user_1.md

```
@@ -0,0 +1 @@
+Given project "{{triggerNode_1.selectedIdea}}" with this technical blueprint: {{triggerNode_1.blueprint}}, and a timeline of {{triggerNode_1.duration}}, generate: a week-by-week roadmap (array of week number and task), an abstract of about 150 words, 5 viva questions ranging from beginner to advanced, and 3 ATS-friendly resume bullet points.
```

markdownlint-cli2 (0.23.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)